modules/apps/29-fee/ibc_middleware.go

```diff
@@ -248,7 +248,7 @@ func (im IBCMiddleware) OnAcknowledgementPacket(
 		return im.app.OnAcknowledgementPacket(ctx, packet, acknowledgement, relayer)
 	}
 
-	var ack = &types.IncentivizedAcknowledgement{}
+	ack := &types.IncentivizedAcknowledgement{}
```

Comment:
Could you please make here the same modification that @damiannolan [requested in the previous PR](https://github.com/cosmos/ibc-go/pull/1717#discussion_r923161753)?

Reply:
happy to!  Will neovim it up. 

Would you mind explaining to me the difference?

Reply:
ah-- these are different

<img width="675" alt="Screen Shot 2022-07-21 at 7 58 08 AM" src="https://user-images.githubusercontent.com/7142025/180107181-68cf3bd6-b9b4-45a3-b71e-ccf0f77c6f31.png">

Here it is:
```go
var ack = &types.IncentivizedAcknowledgement{}
ack := &types.IncentivizedAcknowledgement{}
```

There it was:

```go
var ack = &types.IncentivizedAcknowledgement{}
ack := &types.IncentivizedAcknowledgement
```

I don't mind making the change at all, and am instead curious if these are differences in **style** or differences in **function**.

Thanks!

Reply:
hehehe no I totally understand.

I should close this PR, and go back over to the linting one.

Reply:
ohhhhhh

hence you included the second line.  THanks!